materialize-snowflake/snowflake.go

```diff
@@ -523,4 +524,10 @@ func (f *scratchFile) put(ctx context.Context, conn *stdsql.Conn, cfg *config) e
 	return nil
 }
 
-func main() { boilerplate.RunMain(newSnowflakeDriver()) }
+func main() {
+	// gosnowflake also uses logrus for logging and the logs it produces may be confusing when
+	// intermixed with our connector logs. We disable the gosnowflake logger here and log as needed
+	// when handling errors from the sql driver.
+	sf.GetLogger().SetOutput(io.Discard)
```

Comment:
I'm wondering if we can instead do
```
var logger = log.New()
sf.GetLogger().SetOutput(logger.Writer())
```

Ideally we would have the same instance of `logger` as the rest of the code so it has the same settings, but the idea is instead of discarding these logs, send them to logrus `io.Writer`

Reply:
Hmm I tried this and it still logs as an `ERROR` level. I think the `Writer()` is set at `info` level, but the logs it writes are still at the level they are logged at - just that it won't log anything below `info`. Aside from that, the internal logs from the `gosnowflake` driver aren't very user friendly and might be confusing for a user so its nice to completely suppress them.